README.md

```diff
@@ -1 +1,271 @@
-# Microverse-hello
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
```

Comment:
- [ ] I will advise you to remove this Microverse dummy data and add yours if you have one, if you don't, just remove it and add the name of the project.